### ISSUE TITLE
Text feature issue

### ISSUE THREAD

**Issue by tohich51** [catboost/catboost#1477]
Problem: cb can’t handle text feature, suppose the problem caused by html tags
catboost version: 0.24.2
Operating System: macOS 10.15.7
CPU: 2,6 GHz 6-Core Intel Core i7

`CatBoostError: Invalid type for text_feature[non-default value idx=0,feature_idx=0]="I thought you were in a meeting--? <BR> I am.  With you." : text_features must have string type`

**Comment by feeeper** [comment]
Could you please provide reproducable code sample? It would be great if you add test dataset with the "invalid" text.

**Comment by selfcontrol7** [comment]
Hi,
I have the same error, so I thought opening a new issue wouldn't help.

My inputted text is a Korean text.
Can text features handle strings other than Romanize text? Like Japanese, Chinese, Korean?

Just in case here is a reproducible code sample after reading the train and test dataset from csv file:
```
df_train['Transcript'] = df_train['Transcript'].astype(str)
X_train = df_train.Transcript.tolist()

y_train, y_test = df_train['Label'], df_test['Label']

X_train_text, X_valid_text, y_train_sub, y_valid = train_test_split(X_train, y_train, test_size=0.1)

# Testing the CatBoost text feature
model = CatBoostClassifier(n_estimators=2000,
                           text_features=[0],
                           colsample_bylevel=0.06,
                           max_leaves=31,
                           subsample=0.67,
                           verbose=0,
                           thread_count=6,
                           random_state=1234)

model.fit(X_train_text,
          y_train_sub,
          eval_set = [(X_valid_text, y_valid)],
          early_stopping_rounds=15,
          verbose=0,
          plot=True)
```

Thank you.

**Comment by andrey-khropov** [comment]
This issue has been fixed in CatBoost 0.25.

> Can text features handle strings other than Romanize text? Like Japanese, Chinese, Korean?

Yes, CatBoost fully supports Unicode. If you load data [from delimiter-separated values files](https://catboost.ai/en/docs/concepts/input-data_values-file) use utf-8 encoding in these files.
